### PR TITLE
Add cassandra_system log filter

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

**Comment by eherot** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;cVv3ql3K&#x2F;1195-2-cassandra-logs-to-elk"> (2) Cassandra logs to elk</a>

File: files/default/filter_cassandra_system [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# This file is managed by Chef
+
+filter {
+  if [type] == "cassandra_system" {
+    grok {
+      match => {
+        "message" => [
+          "%{WORD:level}\s+\[%{DATA:thread}\]\s+(?<logtime>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3})\s+%{JAVACLASS:srcclass}:\d+ - %{GREEDYDATA:data}", "%{WORD:level}\s+\[%{DATA:thread}\]\s+(?<logtime>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3})\s+%{DATA:worker}\s+%{JAVACLASS:srcclass}:\d+ - %{GREEDYDATA:data}"
```
Comment (jeffbyrnes) [inline files/default/filter_cassandra_system]: This is some epic RegEx.

**Comment by jeffbyrnes** [comment]
LGTM :shipit:

File: files/default/filter_cassandra_system (hunk 1 above)
Comment (eherot) [inline files/default/filter_cassandra_system]: Welcome to logstash.

**Pull request merged**